spring-cloud-dataflow-server-core/src/main/resources/org/springframework/cloud/dataflow/server/db/migration/h2/V1__INITIAL_SETUP.sql

```diff
@@ -37,12 +37,14 @@ create table audit_records (
 create table stream_definitions (
   definition_name varchar(255) not null,
   definition clob,
+  description varchar(255),
```

Comment:
Actual database migrations are in https://github.com/spring-cloud/spring-cloud-dataflow/tree/master/spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/db/migration and this would result a new schema version. With embedded we can really just keep existing V1 schema as DB is always created from scratch when server starts.

Also those old `1.7.x` migration scripts are not used anymore as things are handled via flyway.